src/admin-views/dashboard/endpoints/endpoint.php

```diff
@@ -19,88 +19,109 @@
  */
 ?>
 
-<div
-	class="tec-automator-grid tec-automator-grid-row tec-automator-endpoint-dashboard-grid tec-automator-settings-details__container tec-settings-connection-endpoint-dashboard-details__container"
-	data-endpoint-id="<?php echo esc_attr( $endpoint['id'] ); ?>"
->
+<div class="tec-automator-grid tec-automator-grid-row tec-automator-endpoint-dashboard-grid tec-automator-settings-details__container tec-settings-connection-endpoint-dashboard-details__container" data-endpoint-id="<?php echo esc_attr( $endpoint['id'] ); ?>">
 	<?php
-		$this->template( 'components/read-only', [
-			'classes_wrap'  => [ 'tec-automator-grid-item', 'tec-settings-connection-endpoint-dashboard-details__name-wrap', ! $endpoint['enabled'] || $endpoint['missing_dependency'] ? 'disabled' : '', ],
+	$this->template(
+		'components/read-only',
+		[
+			'classes_wrap'  => [ 'tec-automator-grid-item', 'tec-settings-connection-endpoint-dashboard-details__name-wrap', ! $endpoint['enabled'] || $endpoint['missing_dependency'] ? 'disabled' : '' ],
```

Comment:
This is super long. It might be better to do this:
```suggestion
			'classes_wrap'  => [
				'tec-automator-grid-item',
				'tec-settings-connection-endpoint-dashboard-details__name-wrap',
				! $endpoint['enabled'] || $endpoint['missing_dependency'] ? 'disabled' : '',
			],
```